src/main/java/Delete.java

```diff
@@ -0,0 +1,147 @@
+                        "\t____________________________________________________________\n");
+            } else if (line.equals("list")) {
+                System.out.print(
+                        "\t____________________________________________________________\n" +
+                        "\tHere are the tasks in your list:\n");
+                int number = 1;
+                for (Task t : tasks) {
+                    System.out.print("\t" + number + "." + t + "\n");
+                    number++;
+                }
+                System.out.print(
+                        "\t____________________________________________________________\n");
+            } else if (line.startsWith("done")) {
+                int num = Integer.parseInt(line.substring(line.indexOf(" ")+1));
+                tasks.get(num-1).getDone(true);
+                System.out.println(
+                        "\t____________________________________________________________\n" +
+                        "\tNice! I've marked this task as done: \n" + "\t" +
+                        tasks.get(num-1)  + "\n" +
+                        "\t____________________________________________________________\n");
+            }  else if (line.startsWith("delete")) {
+                int num = Integer.parseInt(line.substring(line.indexOf(" ") + 1));
+                Task item = tasks.get(num - 1);
+                tasks.remove(item);
+                System.out.println(
+                        "\t____________________________________________________________\n" +
+                        "\tNoted. I've removed this task: \n" +
+                        "\t" + item + "\n" +
+                        "\tNow you have " + tasks.size() + " tasks in the list.\n" +
+                        "\t____________________________________________________________");
+            } else if (line.equals("bye")){
+                System.out.print(
+                        "\t____________________________________________________________\n" +
+                        "\tBye. Hope to see you again soon!\n" +
+                        "\t____________________________________________________________\n");
+                System.exit(0);
+            } else {
+                System.out.print(
+                        "\t____________________________________________________________\n" +
+                        "\t☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n" +
+                        "\tPlease start with the following title:\n\t[T][ ]todo\n\t[D][ ]deadline\n\t[E][ ]event\n" +
+                        "\t____________________________________________________________\n");
+            }
+            Files.delete(Paths.get("D:/OneDrive - National University of Singapore/Study/NUS/Year 2/Semester 2/CS2101/ip/list.txt"));
+            FileWriter list = new FileWriter("D:/OneDrive - National University of Singapore/Study/NUS/Year 2/Semester 2/CS2101/ip/list.txt",true);
```

Comment:
careful of hard coding paths, this exposed your file structure and will not work if exported via JAR

Use relative paths instead